src/components/Header.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import '../styles/Header.css';
+import { Link } from 'react-router-dom';
+
+const Header = (props) => {
+  return (
+    <div className='header'>
+      <div className='home-btn-area'>
+        <Link to='/'>
+          <button className='home-btn' onClick={() => props.clearSearchGame()}>Home</button>
+        </Link>
+      </div>
+      <div className='title-area'>
+        <h1>Video Game Showcase</h1>
+      </div>
+      <div className='search-area'>
+        <form>
+          <input className='search' type='text' placeholder='search by title' value={props.query} onChange={(event) => props.searchGame(event)}></input>
```

Comment:
Maybe break out the search bar  into a separate component, so it's not included as part of the header in game details page? As a user I would find the search bar in the details page confusing since it doesn't return anything when I interact with it.